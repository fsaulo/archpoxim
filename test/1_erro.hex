0xDC000007
0x00000000
0x00000000
0x00000000
0x00000000
0x00000000
0x00000000
0x00000000
0x00200002
0x00400003
0x54410000
0x03C07FFC
0x28000000
0x28020001
0x28020040
0x28000883
0x280007C0
0x2C000000
0x2C0007C0
0x2C0100C2
0xF0F0F0F0
0xFC000000
