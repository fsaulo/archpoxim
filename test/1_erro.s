.text
	init:
		bun main
		.align 5
	main:
		mov r1, 2
		mov r2, 3
		divi r2, r1, 0
		mov sp, 0x7FFC
		push r0
		push r0, r1, r2
		push r1, r0, r2
		push r2, r3, r0, r1
		push sr
		pop r0
		pop sr
		pop r3, r2, r1, r0, r0
		.4byte 0xF0F0F0F0
		int 0
