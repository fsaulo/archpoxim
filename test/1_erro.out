[START OF SIMULATION]
0x00000000:	bun 7                    	PC=0x00000020
0x00000020:	mov r1,2                 	R1=0x00000002
0x00000024:	mov r2,3                 	R2=0x00000003
0x00000028:	divi r2,r1,0             	R2=R1/0x00000000=0x00000003,SR=0x00000020
0x0000002C:	mov sp,32764             	SP=0x00007FFC
0x00000030:	push -                   	MEM[0x00007FFC]{}={}
0x00000034:	push -                   	MEM[0x00007FFC]{}={}
0x00000038:	push r1                  	MEM[0x00007FFC]{0x00000002}={R1}
0x0000003C:	push r2,r3               	MEM[0x00007FF8]{0x00000003,0x00000000}={R2,R3}
0x00000040:	push sr                  	MEM[0x00007FF0]{0x00000020}={SR}
0x00000044:	pop -                    	{}=MEM[0x00007FEC]{}
0x00000048:	pop sr                   	{SR}=MEM[0x00007FEC]{0x00000020}
0x0000004C:	pop r3,r2,r1             	{R3,R2,R1}=MEM[0x00007FF0]{0x00000000,0x00000003,0x00000002}
[INVALID INSTRUCTION @ 0x00000050]
[END OF SIMULATION]
